raimitigations/dataprocessing/imputer/imputer.py

```diff
@@ -56,10 +63,31 @@ def _set_column_to_impute(self):
 
         self.print_message(
             f"No columns specified for imputation. These columns "
-            + f"have been automatically identified:\n{col_with_nan}"
+            + f"have been automatically identified at fit time:\n{col_with_nan}"
         )
 
+        self.col_impute = col_with_nan
+
+    # -----------------------------------
+
+    def _reset_columns_to_impute(self, df: Union[pd.DataFrame, np.ndarray]):
+        if self.none_status is True:
```

Comment:
Just a small comment: in this method, if ``not self.none_status``, then you shouldn't do anything. So, instead of doing ``If something is True: <do a bunch of stuff>``, switch this to: ``if not something: return``.

The way it is, the entire method is inside an ``if`` statement. The other way, you just return if ``not self.none_status``, or do whatever you need to do otherwise, outside any ``if`` statement.

Reply:
done!